### PR TITLE
Add DKIM config example

### PR DESCRIPTION
**Pull request by SorinGFS**
### What does it do?

Updating the `readme` file by adding the DKIM configuration example for `strapi-provider-email-sendmail`.

### CODE REVIEW

File: packages/strapi-provider-email-sendmail/README.md [hunk 1]
```diff
@@ -51,3 +52,36 @@ module.exports = ({ env }) => ({
...
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    provider: 'sendmail',
+    providerOptions: {
+      dkim: {
+        privateKey: 'replace-with-dkim-private-key',
+        keySelector: 'abcd', // the same as the one set in DNS txt record, use online dns lookup tools to be sure that is retreivable
```
Comment (derrickmehaffy) [inline packages/strapi-provider-email-sendmail/README.md:77]: To merge a PR like this, these steps need to be documented somewhere. @pwizla how should we handle this as it falls in the topic of moving provider documentation to the actual documentation?

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]
Need information from the documentation team regarding additional documentation and migration of readme docs to the vuepress docs.

**Review by derrickmehaffy — APPROVED** [review]
LGTM but I do thing we need something in the documentation about this. Larger topic though as we should move much of these instructions from package read-mes to our actual documentation.

File: packages/strapi-provider-email-sendmail/README.md (hunk 1 above)
Comment (pwizla) [reply packages/strapi-provider-email-sendmail/README.md:77]: I totally missed this notification, Derrick, sorry!

I need to think about it and probably discuss it with Mégane & Jim. 
Adding this to my todolist, thanks for your patience.

**Pull request merged**